packages/eslint-plugin-pf-codemods/index.js

```diff
@@ -23,6 +23,7 @@ const warningRules = [
   "overflowMenu-warn-updated-dropdownItem",
   "popover-warn-appendTo-default",
   "react-dropzone-warn-upgrade",
+  "react-icons-warn-remove-props",
```

Comment:
i think to @wise-king-sullyman 's point, this should probably be an error. these properties were removed

Reply:
oh... was i wrong? would this not be an error in the code?

Reply:
@gitdallas I think either would make sense here - should it be a warning since we don't specifically provide the fix? Or can it be an error even if we don't provide the fix